test/fw/ptl/lib/pbs_testlib.py

```diff
@@ -5248,19 +5250,28 @@ def save_configuration(self, outfile, mode='a'):
                 qmgr, '-c', 'print server'], sudo=True,
             logerr=False, level=logging.DEBUG)
         if ret['rc'] != 0:
+            self.logger.error('Failed to get Server attributes')
             return False
         else:
             conf['qmgr_print_server'] = ret['out']
         ret = self.du.run_cmd(self.hostname, [qmgr, '-c', 'print sched'],
                               logerr=False, level=logging.DEBUG, sudo=True)
         if ret['rc'] != 0:
+            self.logger.error('Failed to get sched attributes')
             return False
         else:
             conf['qmgr_print_sched'] = ret['out']
         ret = self.du.run_cmd(self.hostname, [pbsnodes, '-av'],
                               logerr=False, level=logging.DEBUG)
-        nodes_val = self.utils.convert_to_dictlist(ret['out'])
-        conf['pbsnodes'] = nodes_val
+        err_msg = "Server has no node list"
+        # pbsnodes -av returns a non zero exit code when there are
+        # no nodes in cluster
+        if ret['rc'] != 0 and err_msg in ret['err']:
+            self.logger.error('Failed to nodes info')
```

Comment:
type: "failed to get nodes info" instead of "Failed to nodes info"

Reply:
Changed